### ISSUE TITLE
Replace url in protocol definition package storage.ts - find alternative

### ISSUE THREAD

**Issue by sdeshpande3** [microsoft/FluidFramework#8277]
The `url` property for the `ICreateBlobResponse` interface has been deprecated (`\common\lib\protocol-definitions\src\storage.ts`).

There is currently no replacement proposed for this property, but this property is also unused at the moment*.

**Comment by ssimic2** [comment]
@jatgarg can you comment on the current state in regard to this ticket? I see you have moved the field back here: https://github.com/microsoft/FluidFramework/pull/7806/files with some comments here: https://github.com/microsoft/FluidFramework/pull/7774. Could you point us here towards tickets/discussions around mfs-loader dependency ?

**Comment by jatgarg** [comment]
@vladsud Could you point us to people owning/handling mfs-loader so that we can confirm whether they have bumped the dep or not. Yes nobody uses it and we can remove it but it can create the same problem again in this case. Also there might be other packages which will still depend on older package and could cause same issue. Can you please suggest what is the right approach here.

**Comment by vladsud** [comment]
I have not looked at details here, but I think we should do proper deprecation, given partners couple releases to react, and then be free to remove deprecated functionality. Waiting for particular partner(s) does not scale well.
Taht said, we should do so with understanding that certain things (loader versions) live really long time. So, if deprecation touched compat boundary that is has really long SLA (like loader - runtime boundary), then removal can't be done quickly.

**Comment by ssimic2** [comment]
@jatgarg I do not see direct dependency mfs-loader had on `@fluidframework/protocol-definitions`, even back in Sep/Oct. Let's chat about this offline just to be clear about the root cause here.

@vladsud looking at Kusto telemetry, the lowest loader version logged during the last 60 days was 0.45.2. And all of this was "Demo Host" which is our tests.  Beyond that, the lowest version out there is 0.46.0. I'll double check this data, but yes for loader - runtime boundary compatibility reference, we are considering LTS (^0.45.0), which (it looks like) you bumped few months ago.

**Comment by ssimic2** [comment]
@jatgarg I don't see direct dependency mfs-loader has on protocol-definitions. Or OWH having direct dependency on mfs-loader.

To run a quick test, I picked bohemia commit where Kenneth moved protocol-definitions to ^0.1025.1 (url change reverted in protocol definitions), then I pinned protocol-definitions to 0.1025.0 across entire project (0.1025.0 having url removed in protocol definitions). I did not see build errors.

I'll check with Kenneth and Kenny Still tomorrow. If no further info, I will send through the same changes that we needed to revert earlier. If there is a bug, next time around, let's make sure we document repro steps.
